percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.24-25.11.rst

```diff
@@ -0,0 +1,54 @@
+.. rn:: 5.6.24-25.11
+
+=======================================
+ |Percona XtraDB Cluster| 5.6.24-25.11 
+=======================================
+
+Percona is glad to announce the release of |Percona XtraDB Cluster| 5.6 on June 3rd 2015. Binaries are available from `downloads area <http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/release-5.6.24-25.11/>`_ or from our :doc:`software repositories </installation>`.
+
+Based on `Percona Server 5.6.24-72.2 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.24-72.2.html>`_ including all the bug fixes in it, *Galera Replicator* 3.11 and on *Codership wsrep API* 25.11 is now the current **General Availability** release. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.24-25.11 milestone <https://launchpad.net/percona-xtradb-cluster/+milestone/5.6.24-25.11>`_ at Launchpad.
```

Comment:
Need to link to galera and codership milestones: https://github.com/codership/galera/milestones/25.3.11 and  https://github.com/codership/mysql-wsrep/milestones/5.6.x-25.11

However, they are not complete yet. So, I suggest mentioning those milestones and adding 'based on' (or a variant of it).